deployment/common/changeset/link_transfer_timelock.go

```diff
@@ -0,0 +1,177 @@
+package changeset
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	owner_helpers "github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	chain_selectors "github.com/smartcontractkit/chain-selectors"
+
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+)
+
+type LinkTransfer struct {
+	To    common.Address
+	Value *big.Int
+}
+type LinkTransferTimelockConfig struct {
+	Transfers  map[uint64][]LinkTransfer
+	UseMCMS    bool
```

Comment:
don't need the bool anymore. If MCMSConfig is nil consider it false

Reply:
fixed here https://github.com/smartcontractkit/chainlink/pull/15512/commits/6641f4f9112ee08257fa68e04bcc8d751cc649d8